Bias-Correction/bias_correction.ipynb

```diff
@@ -54,28 +35,39 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
-    "hist_file = '../../Seth Comparison/uas.hist.CanESM2.CRCM5-UQAM.day.NAM-44i.raw.nc'\n",
-    "ds_hist = xr.open_dataset(hist_file)"
+    "hist_file = DATASETS.fetch(\"uas.hist.CanESM2.CRCM5-UQAM.day.NAM-44i.raw.Colorado.nc\")\n",
+    "rcp85_file = DATASETS.fetch(\"uas.rcp85.CanESM2.CRCM5-UQAM.day.NAM-44i.raw.Colorado.nc\")\n",
+    "meas_file = DATASETS.fetch(\"uas.gridMET.NAM-44i.Colorado.nc\")"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "# Or if not on Cheyenne\n",
+    "hist_file = DATASERTS.fetch(\"/uas.hist.CanESM2.CRCM5-UQAM.day.NAM-44i.raw.Colorado.nc\")\n",
+    "rcp85_file = DATASERTS.fetch(\"/uas.rcp85.CanESM2.CRCM5-UQAM.day.NAM-44i.raw.Colorado.nc\")\n",
+    "meas_file = DATASETS.fetch(\"uas.gridMET.NAM-44i.Colorado.nc\")"
```

Comment:
Can we remove this cell from the notebook?

Reply:
removed